config/initializers/rswag-ui.rb

```diff
@@ -0,0 +1,14 @@
+Rswag::Ui.configure do |c|
```

Comment:
is it a typo that one is rswag_api with underscore and this one is with a dash?

Reply:
Pretty sure this was generated automatically by a rake task as well. We can name them whatever we like though. I guess the underscore is a bit more consistent with our general naming schemes. I'll update it :+1: